src/Plugin/Patches.php

```diff
@@ -443,6 +443,14 @@ protected function getAndApplyPatch(RemoteFilesystem $downloader, $install_path,
             // differences between how patch works on windows and unix.
             $patch_options = '--no-backup-if-mismatch';
             if (PHP_OS_FAMILY == 'BSD') {
+                $patchVersion = '';
+                $this->executor->execute('patch --version', $patchVersion);
+                if (strpos($patchBinaryVersion, 'GNU patch') === false) {
+                    throw new \Exception(
```

Comment:
The intention here should be for non-GNU versions of patch to actually work! :)  Please review the PR at #402 where we try to get the non-GNU patch working (with a pretty trivial fix).